builders/ethereum/dev-env/thirdweb.md

```diff
@@ -56,7 +56,7 @@ To create a new smart contract using the [thirdweb CLI](https://portal.thirdweb.
     - Inheriting the contract by declaring that your contract is an `ERC721Base` contract
     - Implementing any required methods, such as the constructor
 
-5. After modifying your contract with your desired custom logic, you can deploy it to Moonbeam using [Deploy](https://portal.thirdweb.com/contracts/deploy/overview){target=\_blank}. That will be covered in the next section!
+5. After modifying your contract with your desired custom logic, you can deploy it to Moonbeam using [Deploy](https://portal.thirdweb.com/contracts#deploy-a-contract){target=\_blank}. That will be covered in the next section!
```

Comment:
Seems like they don't call it "Deploy" like they used to. Maybe need to change all instances of Deploy to contract deploy API